pkg/config/apiserver.go

```diff
@@ -88,6 +104,58 @@ func (a *API) register(c *restful.Container) {
 		Param(ws.PathParameter("subject", "subject").DataType("string")).
 		Reads(&pb.ServiceConfig{}).
 		Writes(APIResponse{}))
+
+	// Creates or replaces a rule's list of aspects
+	ws.Route(ws.
+		PUT("/scopes/{scope}/subjects/{subject}/rules/{ruleid}/aspects/{aspect}").
+		To(a.putAspect).
+		Doc("Creates or replaces a rule’s list of aspects.").
+		Param(ws.PathParameter("scope", "scope").DataType("string")).
+		Param(ws.PathParameter("subject", "subject").DataType("string")).
+		Param(ws.PathParameter("aspect", "aspect").DataType("string")).
```

Comment:
I missed the ruleid parameter; good catch.  Regarding rule_id vs ruleid, I am following _istio/api/mixer/v1/config_api_swagger.yaml_:124-156 which uses ruleid.